### PR TITLE
Create esim voucher apip 605

### PR DESCRIPTION
**Pull request by alexairalo**
https://airalo.atlassian.net/browse/APIP-605

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -541,6 +541,69 @@ Example response:<br>
...
+use Airalo\Airalo;
+use Airalo\AiraloStatic;
+
+$alo = new Airalo([
+    'client_id' => '<YOUR_API_CLIENT_ID>',
+    'client_secret' => '<YOUR_API_CLIENT_SECRET>',
+]);
+
+
+$vouchers = $alo->esimVouchers([
```
Comment (MiroIsikiyski) [inline README.md:563]: is it possible to have the function signature to be more consistent with the rest of the methods, for example quantity is separate parameter for orders, can we do the same here?

**Review by raza-airalo — APPROVED** [review]

**Review by AndreasMaalouf — APPROVED** [review]

File: src/Constants/ApiConstants.php [hunk 2]
```diff
@@ -20,6 +20,8 @@ final class ApiConstants
 
     public const VOUCHERS_SLUG = 'voucher/airmoney';
 
+    public const VOUCHERS_ESIM_SLUG = 'voucher/eesim';
```
Comment (MiroIsikiyski) [inline src/Constants/ApiConstants.php:23]: I guess a typo? `eesim` -> `esim`?

File: README.md (hunk 1 above)
Comment (alexairalo) [reply README.md:563]: @MiroIsikiyski basically it's multiple voucher creation so for each esim voucher user can use any quantity, so we cannot add second argument called $quantity. What we can do is to use DTO collection object

File: src/Constants/ApiConstants.php (hunk 2 above)
Comment (alexairalo) [reply src/Constants/ApiConstants.php:23]: @MiroIsikiyski thanks!, fixed

**Pull request merged**